bot/client.js

```diff
@@ -15,6 +15,14 @@ client.on('ready', () => {
   util.log('Bot Online and Ready', 0);
 });
 
+// Message for new users
+client.on('guildMemberAdd', (member) => {
+  // Send Direct Message with Terms of Service
+  member.send(`Welcome ${member} to the Full Sail Armada! Read our Terms of Service`);
+  // Send Welcome User Message in the General Channel
+  member.guild.channels.find('name', 'general').send(`Welcome ${member} to the Full Sail Armada!`);
```

Comment:
When we bury strings inside of code that has been chained, even somewhat non-complex code, we make it hard to maintain the text messages we are trying to send. This is strongly related to [magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544) in that if we implement a variable, and control that string within, using the variable on the sending of the message method will come out cleaner. If you look at the email being sent in the verify controller you can see how quickly inline strings used in this manner begin to degrade.

```javascript
const targetChannel = 'general';
const greeting = 'Welcome ${member.user.username} to the Family!';

// Send Welcome User Message in the General Channel
member.guild.channels.find('name', targetChannel).send(greeting);
```